site/faq.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Weave Net FAQ
+layout: default
+---
+
+
+
+###Q:How do I obtain the IP of a specific container when I'm using Weave?
+
+Use `weave ps` to see which address was allocated to a container on a Weave network.  
+
+See [Address Allocation](/site/features/address-allocation.md).
+
+
+###Q: How do I expose one of my containers to the outside world?
+
+Exposing a container to the outside world is described in [Exposing Services to the Outside](/site/using-weave/service-export.md).
+
+
+###Q: My dockerized app needs to check the request of a remote IP. Can I manually change the IP of a container?
+
+
+You can manually change the IP of a container using [Classless Inter-Domain Routing or CIDR notation](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing). 
+
+For more information, refer to [Manually Specifying the IP Address of a Container](/site/using-weave/manual-ip-address.md). 
+
+
+###Q:Can I connect my existing 'legacy' network with a Weave container network?
+
+Yes you can. 
+
+Let's say you have a Weave network that runs on hosts A, B, C. and an additional host, that we'll call P, where neither Weave nor Docker are running.  You want to connect from a process running on host P to a specific container running on host B on the Weave network. 
```

Comment:
It would be better to start with "run Weave on the host and do `weave expose`".  Only if they _cannot_ run Weave on the host do they need to go with the "host P" scenario.